### PR TITLE
Link-Redirect-fix-DocsHelp-Component

### PR DESCRIPTION
**Pull request by Yashwanth1906**
What kind of change does this PR introduce?
           Change to correct wrong redirecting URLs
Issue Number:
closes: https://github.com/json-schema-org/website/issues/1038

Screenshots/videos:

If relevant, did you update the documentation?

Summary

This pull request addresses an issue related to incorrect URL redirection in the JSON Schema website. Users were being redirected to incorrect or unintended pages when clicking on certain links.

Changes Made:
Corrected Redirecting URLs: Updated the wrong redirecting URLs in various components to ensure they point to the correct destinations. This change enhances navigation and provides users with access to the intended content.

Refactored JSX Components: Adjusted the formatting of JSX components to comply with the project's style guide. Specifically, I ensured that there were proper spaces before closing brackets in the JSX tags, addressing the errors raised by the reviewers.

Code Consistency: Ensured consistency in the codebase by adhering to Prettier and ESLint rules, which included correcting formatting issues to enhance code readability and maintainability.

These changes improve user experience by ensuring that links function correctly, reducing frustration caused by misdirected navigation.

Does this PR introduce a breaking change?

No

### CODE REVIEW

**Comment by Yashwanth1906** [comment]
@DarhkVoyd  This is the new pull request for this closed pull request #1046 because of some git pull rebase issues. I have written all the tests and changed all the corrections that you said in that #1046 closed pull request. Sry for closing that pull request.

**Review by aialok — COMMENTED** [review]
LGTM. Testing part ✅️

**Comment by Yashwanth1906** [comment]
@aialok  Is everything fine with the pr ? Is there any one want to review it?

File: pages/[slug].page.tsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ export default function StaticMarkdownPage({
   frontmatter: any;
   content: any;
 }) {
-  const markdownFile = '_index';
+  const markdownFile = '_md';
```
Comment (DarhkVoyd) [inline pages/[slug].page.tsx:25]:
```suggestion
  const fileRenderType = '_md';
```

**Review by DarhkVoyd — CHANGES_REQUESTED** [review]
Hey, thanks for working on this. This is almost done, just a few suggestions:
1. We can get rid of the `console.log`
2. Since the prop name has been updated to `fileRenderType`, let's update the passed argument variable name to reflect the change. As I have suggested at one of the instances.

**Comment by Yashwanth1906** [comment]
@DarhkVoyd  Done with all the changes suggested.

**Comment by DarhkVoyd** [comment]
@Yashwanth1906 Hey, could you please resolve the conflicts in the PR? I'll review this PR soon.

**Comment by Yashwanth1906** [comment]
@DarhkVoyd ok Ill finsih this as soon as possible

**Comment by DarhkVoyd** [comment]
Hey @Yashwanth1906 You don't have to close existing PR and create a new PR for each change/review, you can continue working on the same branch and PR, we'll squash merge it.

**Comment by Yashwanth1906** [comment]
@DarhkVoyd ok

**Comment by Yashwanth1906** [comment]
@DarhkVoyd Hi, I have tried pulling and merging the files and it resulted in mess and i added some packages and now im superconfused how to resolve this thing. Could I make another clean PR with all the tests and changes that you told in this PR.

**Pull request closed without merging**